### DISCUSSION TITLE
Directly access a tile image file

### DISCUSSION THREAD

**Discussion by timbtimbtimb** [rnmapbox/maps#3628] [Q&A]
Is it possible to access a RasterLayer's tile's PNG file directly? I'd like to make an eye-dropper feature that I have on my website, which gets the pixel color at the center of the map by calulating over which tile the center of the map is, and over which of this tile's pixel.

**Comment by Gnative** [comment]
Hmm that could be complicated, you could look at taking a snapshot with  https://github.com/rnmapbox/maps/blob/main/docs/snapshotManager.md and referencing that. 

Might even be simpler to implement.
